ntoskrnl/kdbg/kdb_cli.c

```diff
@@ -3007,15 +3008,16 @@ KdbpPager(
         {
             while ((p2 = strrchr(p, '\x1b'))) /* Look for escape character */
             {
+                size_t len = strlen(p2);
                 if (p2[1] == '[')
                 {
                     j = 2;
                     while (!isalpha(p2[j++]));
-                    strcpy(p2, p2 + j);
+                    memmove(p2, p2 + j, len + 1 - j);
```

Comment:
Maybe something like:
`memmove(p2, p2 + j, BufLength - (p2 - Buffer));`
may work

Reply:
again, I don't see how that is better.